twitter/twitter_records.bal

```diff
@@ -0,0 +1,703 @@
+# + profile_background_image_url - A HTTP-based URL pointing to the background image the user has uploaded for
+#                                  their profile.
+# + profile_background_image_url_https - A HTTPS-based URL pointing to the background image the user has uploaded
+#                                        for their profile.
+# + profile_background_tile - When true, indicates that the user’s profile_background_image_url should be
+#                             tiled when displayed.
+# + profile_banner_url - The HTTPS-based URL pointing to the standard web representation of the user’s uploaded
+#                        profile banner.
+# + profile_image_url - A HTTP-based URL pointing to the user’s profile image.
+# + profile_image_url_https - A HTTPS-based URL pointing to the user’s profile image.
+# + profile_link_color - The hexadecimal color the user has chosen to display links with in their Twitter UI.
+# + profile_sidebar_border_color - The hexadecimal color the user has chosen to display sidebar borders with in
+#                                  their Twitter UI.
+# + profile_sidebar_fill_color - The hexadecimal color the user has chosen to display sidebar backgrounds with
+#                                in their Twitter UI.
+# + profile_text_color - The hexadecimal color the user has chosen to display text with in their Twitter UI.
+# + profile_use_background_image - When true, indicates the user wants their uploaded background image to be used.
+# + has_extended_profile - When true, indicates that the user has extended their user profile.
+# + default_profile - When true, indicates that the user has not altered the theme or background of their user profile.
+# + default_profile_image - When true, indicates that the user has not uploaded their own profile image and a
+#                           default image is used instead.
+# + follow_request_sent - Whether the authenticating user has requested to follow this user or not.
+# + following - When true, indicates that the authenticating user is following this user.
+# + live_following -
+# + notifications - Indicates whether the authenticated user has chosen to receive this user’s Tweets by SMS.
+# + muting - When true, indicates that the authenticating user has been muted.
+# + blocking - When true, indicates that the authenticating user has been blocked.
+# + blocked_by - When true, indicates that the authenticating user has blocked you.
+# + status - Returns the current status of the user.
+# + translator_type - Type of the translator.
+# + derived - Collection of Enrichment metadata derived for user.
+# + withheld_in_countries - When present, indicates a list of uppercase two-letter country codes
+#                           the content is withheld from.
+# + withheld_scope - When present, indicates that the content being withheld is a "user".
+public type User record {
+    int id;
+    string id_str;
+    string name;
+    string screen_name;
+    string? location;
```

Comment:
Is this `location` a nillable field or optional field?
`string? location` vs `string location?`

Shall we rethink and fix that for other fields as well?

Reply:
It is a Nullable param as per this doc

https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/user-object